test/tests/functional/pbs_node_buckets.py

```diff
@@ -695,6 +695,10 @@ def test_psets_spanning(self):
         Request more nodes than available in one placement set and see
         the job span or not depending on the value of do_not_span_psets
         """
+        # Turn off scheduling to be sure there is no cycle running when
+        # configurations are changed
+        a = {'scheduling': 'False'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
         a = {'node_group_key': 'shape', 'node_group_enable': 'True'}
```

Comment:
combine the two manager calls?

Reply:
You are right these two calls can be combined, but I'd like to retain if you think the impact of this code is not huge. The reason is that this bug is a timing issue and occurs once in a while which makes verification of this bug difficult. I have to run the same testsuite on a slow machine 20 times to make sure it is fixed which takes about 15hrs.